.github/workflows/build-verification.yml

```diff
@@ -25,3 +25,69 @@ jobs:
         run: ./gradlew build -i
         env:
           GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_SOLUTIONS_ACCESS_TOKEN }}
+      - name: Extract build validation Scripts
```

Comment:
I would make the new stuff here at least a separate job. Possibly, run sequentially after the `Verification` job.